blackduck/Client.py

```diff
@@ -56,33 +56,41 @@ def __init__(self, base_url, timeout, retries, verify):
 
     def request(self, method, url, **kwargs):
         kwargs['timeout'] = self._timeout
+        if method.lower() == 'get':
+            # set default media type if not provided
+            headers = {
+                'accept': "application/json",
+                'content-type': "application/json"
+            }
+            headers.update(kwargs.pop('headers', dict()))
+            kwargs['headers'] = headers
         url = urljoin(self.base_url, url)
         return super().request(method, url, **kwargs)
```

Comment:
What is super calling here? Client does/did not have any parent class.

Reply:
This is actually in HubSession within Client.py. So it is calling requests.session.request.

Reply:
Ah I see - apologies.